zipkin-server/server-core/src/main/java/zipkin/server/core/CoreModuleProvider.java

```diff
@@ -133,8 +152,8 @@ public void prepare() throws ServiceNotProvidedException, ModuleStartException {
     final WorkerInstancesService instancesService = new WorkerInstancesService();
     this.registerServiceImplementation(IWorkerInstanceGetter.class, instancesService);
     this.registerServiceImplementation(IWorkerInstanceSetter.class, instancesService);
-    this.registerServiceImplementation(RemoteSenderService.class, new RemoteSenderService(getManager()));
-    this.registerServiceImplementation(RemoteSenderService.class, new RemoteSenderService(getManager()));
+    // no cluster mode for zipkin, for sending the streaming data to the local
+    this.registerServiceImplementation(RemoteSenderService.class, new SelfSenderService(getManager()));
```

Comment:
I think we still need cluster mode due to TTL task.

Reply:
Sure, I will create other PR to add the cluster mode.